index.html

```diff
@@ -8,6 +8,7 @@
 
   <body>
     <div id="mount"></div>
+    <script>window.debuggerConfig = <%= debuggerConfig %>; </script>
```

Comment:
The only problem with this is that we aren't running a server in the devtools panel, so this won't work. We need raw HTML file for the panel.